index.bs

```diff
@@ -138,8 +147,29 @@ intersection with the <a>intersection root</a>, as per the
 The IntersectionObserver interface</h3>
 
 The {{IntersectionObserver}} interface can be used to observe changes in the
-intersection of a <a>target</a> {{Element}} and a {{IntersectionObserver/root}}
-{{Element}} (or the top-level document's viewport).
+intersection of a {{IntersectionObserver/root}} {{Element}} (or the
+<a>top-level browsing context</a>'s visual viewport), and one or more <a>target</a> {{Element}}s.
+
+An {{IntersectionObserver}} with a {{IntersectionObserver/root}} {{Element}} can
+observe any <a>target</a> {{Element}} that is a descendant of the
+{{IntersectionObserver/root}} in the <a>node tree</a>, including {{Element}}s inside
+an <a>iframe</a> that belongs to the same <a>unit of related similar-origin browsing contexts</a>
+as the {{IntersectionObserver/root}}.  For example, this is a valid root-target
+relationship:
+
+<div class="example">
+     <pre highlight="html">
+     &lt;div id="root"&gt;
+       &lt;iframe srcdoc="&lt;div id='target'&gt;"&gt;&lt;iframe&gt;
+     &lt;div&gt;
+     </pre>
+</div>
+
+An {{IntersectionObserver}} with no {{IntersectionObserver/root}} {{Element}}
+will automatically observe intersections with the <a>top-level browsing context</a>'s
+visual viewport, and valid <a>targets</a> include any {{Element}} in the
```

Comment:
I'm not sold that we should be changing from layout to visual viewport. I think we should probably VC with me, you and @esprehn to hash it out.